slack_bolt/listener_matcher/builtins.py

```diff
@@ -1,543 +0,0 @@
-# pytype: skip-file
```

Comment:
Making these refactoring as part of future branch will make merging it into main more difficult. These big changes are not necessary for implementing functions. Can you do this separately towards main branch?

Reply:
That makes sense, I've removed these changes ✂️ 

It should also make the PR easier to read